CMakeLists.txt

```diff
@@ -24,9 +24,11 @@ if(CMAKE_COMPILER_IS_GNUCC OR CMAKE_COMPILER_IS_GNUCXX OR
     ${CMAKE_CXX_COMPILER_ID} STREQUAL "Clang")
   set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++17")
   set(CMAKE_CXX_FLAGS_RELEASE "${CMAKE_CXX_RELEASE} -O3 -DNDEBUG")
+  include_directories(/usr/include/python)
```

Comment:
Not sure what's the right way to do that. Python include path is missing by default, so Python.h is not found. Default python3 installation puts it into /usr/include/python3, then I create a symlink manually.

Reply:
Point to /usr/include/python3.6

Reply:
I think /usr/include/python3 is a default symlink, can I use that instead?

Reply:
I don't see a python3.6 on my Ubuntu's so it may not be available on all Ubuntu systems. Do you see a /usr/include/python3.6? Also move this include into the Python specific CMakeLists.txt rather than the global CMakeLists.txt.

Also move the ```-fvisibility``` into the Python specific CMakeLists.txt as well.